src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -19,6 +19,7 @@ optionals+=""
 [[ "${TLS_SKIP_VERIFY}" ]] && optionals+="-tls-skip-verify "
 [[ "${PROXY_TIMEOUT}" ]] && optionals+="-proxy-timeout ${PROXY_TIMEOUT} "
 [[ "${AA_KBC_PARAMS}" ]] && optionals+="-aa-kbc-params ${AA_KBC_PARAMS} "
+[[ "${KBS_CERT}" ]] && optionals+="-kbs-cert ${KBS_CERT} "
```

Comment:
I don't think this will work, with "$KBS_CERT" being a multi-line or even just containing whitespaces, did you test that?